### PR TITLE
Mention Jetstack on the front page

### PR DESCRIPTION
**Pull request by james-w**
Uses the CNCF standard wording for this message.

Also add an about page to the end of the docs.

### CODE REVIEW

File: content/en/docs/aboutjetstack/_index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "About Jetstack"
+linkTitle: "About Jetstack"
+weight: 90
+type: "docs"
+---
+
+cert-manager was originally created by [Jetstack](https://jetstack.io), and
+a lot of the development is still sponsored by us.
```
Comment (meyskens) [inline content/en/docs/aboutjetstack/_index.md:9]: the `by us` sounds weird as it suddenly switches to the first person as jetstack. Could we reword this?

Comment (james-w) [reply content/en/docs/aboutjetstack/_index.md:9]: yeah it is a bit weird, but the rest of the page is in first person:

> We offer:

I will try and rewrite it

Comment (james-w) [reply content/en/docs/aboutjetstack/_index.md]: Updated, PTAL

**Comment by meyskens** [comment]
Not a fan of the layout... will take a look on Monday
(https://deploy-preview-203--cert-manager.netlify.app/)

**Comment by meyskens** [comment]
Forked this PR into #215

**Pull request closed without merging**